### PR TITLE
AJ MP - LINK Symptom Checkers to db & forwards to chat

### PR DESCRIPTION
**Pull request by alexanderjaye**

### CODE REVIEW

File: client/src/Components/Pages/PatientSymptoms/PatientSymptoms.tsx [hunk 1]
```diff
@@ -59,7 +62,7 @@ const PatientSymptoms = (props: Props) => {
             area={view}
             onBackArrowClick={() => handleBackArrowClick()}
           />
-      }
+      }x
```
Comment (PLejay) [inline client/src/Components/Pages/PatientSymptoms/PatientSymptoms.tsx:65]:
```suggestion
      }
```

File: client/src/Contexts/Consultation.context.tsx [hunk 2]
```diff
@@ -103,34 +91,38 @@ const ConsultationContextProvider = (props: Props) => {
...
-        patientId: user!.id,
-        patientNotes: '',
-        doctorId: doctorId
-      }
-      // send to backend
-      mutation.mutate(consultation);
+    const confirmConsultation = (): Promise<void> => {
+      return new Promise((resolve, reject) => {
+        try {
+          const selectedSypmtoms = filterSelectedSymptoms([...physicalSymptoms, ...generalSymptoms]);
```
Comment (PLejay) [inline client/src/Contexts/Consultation.context.tsx:106]:
```suggestion
          const selectedSymptoms = filterSelectedSymptoms([...physicalSymptoms, ...generalSymptoms]);
```

Comment (PLejay) [reply client/src/Contexts/Consultation.context.tsx:106]: This bugs me

**Review by guillemgarciagomez — APPROVED** [review]

**Review by PLejay — CHANGES_REQUESTED** [review]

File: client/src/Contexts/Consultation.context.tsx (hunk 2 above)
Comment (mpatel66) [reply client/src/Contexts/Consultation.context.tsx:106]: lol

**Pull request merged**